Src/Base/AMReX_TinyProfiler.H

```diff
@@ -41,15 +50,49 @@ public:
 
 private:
     //! stats on a single process
+#ifndef AMREX_USE_CUPTI
```

Comment:
Could these be combined into a single version with slight differences if AMREX_USE_CUPTI instead of two completely separate implementations?

Reply:
I've made an effort here (and elsewhere) to eliminate redundancy.